lld/test/MachO/reproduce.s

```diff
@@ -1,4 +1,6 @@
-# REQUIRES: x86, shell
+# REQUIRES: x86_64
```

Comment:
Why did you change it from x86 to x86_64?

Reply:
Probably because I erased the line and rewrote it rather than erasing the shell part and never double checked. Fixed now.